src/dmd/utils.d

```diff
@@ -189,24 +189,24 @@ dg  = Delegate to call afterwards
 Returns:
 The return value of `T`
 */
-auto toCStringThen(alias dg)(const(char)[] src) nothrow
```

Comment:
Why remove `nothrow`?

Reply:
My guess is that it should be inferred, and maybe @jacob-carlborg has a non-`nothrow` delegate.

However we don't have any place in DMD that use exceptions, and IIRC Walter is clearly against it, which is why this is here in the first place.

Reply:
Yeah the only reason I could see to remove it is if it were exposed in dmd as a library.